src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -0,0 +1,182 @@
+// Libraries
+import React, {ChangeEvent, FC, FormEvent, useContext, useReducer} from 'react'
+
+// Components
+import {
+  Overlay,
+  Button,
+  ComponentColor,
+  Form,
+  Grid,
+  Columns,
+  InputRef,
+  TextAreaRef,
+  ButtonType,
+} from '@influxdata/clockface'
+import AnnotationSummaryInput from 'src/annotations/components/annotationForm/AnnotationSummaryInput'
+import AnnotationTimeStartInput from 'src/annotations/components/annotationForm/AnnotationTimeStartInput'
+import AnnotationTimeStopInput from 'src/annotations/components/annotationForm/AnnotationTimeStopInput'
+import AnnotationMessageInput from 'src/annotations/components/annotationForm/AnnotationMessageInput'
+import AnnotationTypeToggle from 'src/annotations/components/annotationForm/AnnotationTypeToggle'
+import AnnotationStreamSelector from 'src/annotations/components/annotationForm/AnnotationStreamSelector'
+
+// Form State
+import {
+  annotationReducer,
+  getInitialAnnotationState,
+  AnnotationType,
+  annotationFormIsValid,
+  Annotation,
+  getAnnotationFromDraft,
+} from 'src/annotations/reducers/annotationReducer'
+
+// Types
+import {AnnotationStream} from 'src/annotations/constants/mocks'
+
+// Contexts
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+
+interface Props {
+  title: 'Edit' | 'Add'
```

Comment:
bit of a red flag here that the typing might be wrong. why isn't the title derived from a boolean or enum?